cmd/minikube/main.go

```diff
@@ -141,7 +141,7 @@ func checkLogFileMaxSize(file string, maxSizeKB int64) bool {
 }
 
 // logFileName generates a default logfile name in the form minikube_<argv[1]>_<hash>_<count>.log from args
-func logFileName(dir string, logIdx int64) string {
+func logFileName(dir string, logIdx int64, nonFlagArgs []string) string {
```

Comment:
Thank you for fixing this @vigothehacker how about adding a unit test with examples and test cases?

Reply:
Implemented TestGeneratedLogs in tests. Please see my review as I had some doubts